pymc_marketing/clv/models/modified_beta_geo.py

```diff
@@ -0,0 +1,316 @@
+#   Copyright 2024 The PyMC Labs Developers
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""Modified Beta-Geometric Negative Binomial Distribution (MBG/NBD) model for a non-contractual customer population across continuous time."""  # noqa: E501
+
+import warnings
+
+import numpy as np
+import pandas as pd
+import pymc as pm
+import pytensor.tensor as pt
+import xarray
+from pymc.distributions.dist_math import check_parameters
+from scipy.special import hyp2f1
+
+from pymc_marketing.clv.models import BetaGeoModel
+from pymc_marketing.clv.utils import to_xarray
+
+
+class ModBetaGeoModel(BetaGeoModel):
+    r"""Also known as the MBG/NBD model.
+
+    Based on [5]_, [6]_, this model has the following assumptions:
```

Comment:
Reference numbers require updating. I would also prefer the docstring preamble follows the conventions of the other CLV models.

Reply:
Updated docstring